libs/langgraph/langgraph/pregel/io.py

```diff
@@ -97,6 +97,9 @@ def __radd__(self, other: dict[str, Any]) -> "AddableUpdatesDict":
         raise TypeError("AddableUpdatesDict does not support right-side addition")
 
 
+EMPTY_SEQ = tuple()
```

Comment:
Should we just make a single empty_seq in an internal _constants file or something

Reply:
I guess so?